perf-tool/src/monitor.cpp

```diff
@@ -26,6 +26,7 @@
 #include <jvmti.h>
 #include <ibmjvmti.h>
 #include <map>
+#include <sstream>
```

Comment:
Let's eliminate this include because it is not needed anymore.

Reply:
o! yes, done.